services/VideoService.js

```diff
@@ -4,6 +4,10 @@ exports.findVideoList = async () => {
   return await Video.find().lean();
 };
 
+exports.findUserVideoList = async (id) => {
```

Comment:
이것도 만약 라우트, 컨트롤러랑 통일한다면 MyVideoList가 좋을 거 같아요!
```suggestion
exports.findMyVideoList = async (id) => {
```